src/html/partials/story/intro.hbs

```diff
@@ -216,47 +225,56 @@ TODO: more examples, quick slideshow. Is this song actually atypical? Why? If no
 
 <div class="step" data-song="Royals">
 <p>
-Lorde burst on the scene in 2013 with her singular debut single, <i>Royals</i>.
```

Comment:
I'm not going to lie, I kind of liked this dumb pun :(

Reply:
Ha, put it back and I'll see if I can get it by. I pulled it only because my brain tripped over it as I was reading and it took me a minute to parse.